.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
  - export M2_HOME=$PWD/apache-maven-3.6.0
  - export PATH=$M2_HOME/bin:$PATH
 script:
- - mvn install
+ - mvn scoverage:report
```

Comment:
Does this do everything that `mvn install` does? (compile+run tests)

Reply:
It looks like it does (https://scoverage.github.io/scoverage-maven-plugin/1.4.0-M5/plugin-info.html).